src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1151,6 +1151,7 @@ export const ComputeModalBase = ({
           'Learn more about autopause.',
           icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
         ]),
+        // end Autopause component
```

Comment:
I mislead you on this comment - the NumberInput component should be included as well